mne/datasets/__init__.py

```diff
@@ -25,6 +25,7 @@
 from . import refmeg_noise
 from . import ssvep
 from . import erp_core
+from . import epilepsy
```

Comment:
I fear that epilepsy is too generic. Can you add ecog to the name? unless we think we will put in this dataset all epilepsy related data?